### PR TITLE
register_satellite.rb - update to find location ID if not provided based on provider location tag

### PR DESCRIPTION
**Pull request by itewk**

### CODE REVIEW

File: Automate/RedHatConsulting_Satellite6/Integration/Satellite/Operations/Methods.class/__methods__/register_satellite.rb [hunk 1]
```diff
@@ -70,15 +77,41 @@ def get_satellite_api()
   mac                       = vm.mac_addresses[0]
   ip                        = nil # TODO: eventully this should come from someplace like Infoblox, for now Satellite assigns it from its DB
   satellite_organization_id = prov.get_option(:satellite_organization_id) || prov.get_option(:ws_values)[:satellite_organization_id]
-  satellite_location_id     = prov.get_option(:satellite_location_id)     || prov.get_option(:ws_values)[:satellite_location_id]
+  satellite_location_id     = prov.get_option(:satellite_location_id)     || prov.get_option(:ws_values)[:satellite_location_id]
```
Comment (henrygrover) [inline Automate/RedHatConsulting_Satellite6/Integration/Satellite/Operations/Methods.class/__methods__/register_satellite.rb]: Random space added?

File: Automate/RedHatConsulting_Satellite6/Integration/Satellite/Operations/Methods.class/__methods__/register_satellite.rb [hunk 2]
```diff
@@ -70,15 +77,41 @@ def get_satellite_api()
...
-  error("Required miq_provision option <satellite_organization_id> not found") if satellite_organization_id.nil?
-  error("Required miq_provision option <satellite_location_id> not found")     if satellite_location_id.nil?
-  error("Required miq_provision option <satellite_hostgroup_id> not found")    if satellite_hostgroup_id.nil?
-  error("Required miq_provision option <satellite_domain_id> not found")       if satellite_domain_id.nil?
+  error("Required miq_provision option <satellite_organization_id> not found") if satellite_organization_id.blank?
+  error("Required miq_provision option <satellite_hostgroup_id> not found")    if satellite_hostgroup_id.blank?
+  error("Required miq_provision option <satellite_domain_id> not found")       if satellite_domain_id.blank?
+  
+  # if satellite location id not set as a provisioning option then
+  #   determine if VM ownning provider has a location tag, and if so, use that
```
Comment (henrygrover) [inline Automate/RedHatConsulting_Satellite6/Integration/Satellite/Operations/Methods.class/__methods__/register_satellite.rb]: Ownning->owning

File: Automate/RedHatConsulting_Satellite6/Integration/Satellite/Operations/Methods.class/__methods__/register_satellite.rb (hunk 1 above)
Comment (itewk) [reply Automate/RedHatConsulting_Satellite6/Integration/Satellite/Operations/Methods.class/__methods__/register_satellite.rb]: fixed

File: Automate/RedHatConsulting_Satellite6/Integration/Satellite/Operations/Methods.class/__methods__/register_satellite.rb (hunk 2 above)
Comment (itewk) [reply Automate/RedHatConsulting_Satellite6/Integration/Satellite/Operations/Methods.class/__methods__/register_satellite.rb]: fixed.

**Comment by itewk** [comment]
@henrygrover fixed issues. re-approve?

**Pull request merged**